app/addons/documents/index-editor/stores.js

```diff
@@ -27,7 +27,7 @@ function (FauxtonAPI, ActionTypes) {
       this._designDocs = [];
       this._view = {
         reduce: this.defaultMap,
-        map: this.defaultReduce
```

Comment:
isn't it tthat reduce must get `this.defaultReduce` instead of `this.defaultMap`?